utility/utility.go

```diff
@@ -99,8 +117,8 @@ func TrimFileExtension(filePath string) string {
 	return strings.TrimSuffix(filePath, "."+GetFileExtension(filePath))
 }
 
-func GetFileRelativePath(fileAbsPath string, directoryPath string) string {
-	return strings.TrimPrefix(fileAbsPath, directoryPath)
```

Comment:
The name is misleading. It only gives a relative path if one path is subpath of another.
I fixed its implementation so that it's more stable and named it accordingly. So what exactly is the problem?